essence-of-live-coding/src/LiveCoding/Forever.lhs

```diff
@@ -141,3 +142,20 @@ Let us run it:
 
 \fxerror{``Forever and ever?'' Show graceful shutdown with ExceptT. Have to change the runtime slightly for this.}
 \fxnote{Awesome idea: Electrical circuits simulation where we can change the circuits live!}
+
+\begin{comment}
+\begin{code}
+foreverCE :: (Monad m, Data e, Finite e) =>
+  e ->
+  (e -> CellExcept a b m e) ->
+  Cell m a b
+foreverCE e f = foreverE e $ commute $ runCellExcept . f
+
+foreverCE' ::
+  (Monad m, Data void, Finite void, Data e, Finite e) =>
```

Comment:
I could avert the constraints on `void` by plugging in `Void` instead. Weird it's necessary at all. Have a look again whether I can get rid of them, and write a test just to see whether they are a nuisance.